tests/test_private_vlan_association.rb

```diff
@@ -0,0 +1,370 @@
+# Copyright (c) 2013-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is there a reason you did not add all of the priviate vlan tests to `test_vlan.rb` given that it's just an extension to that object?  I could probably be convinced that they remain separate, but we should put them all in a single file `test_private_vlan.rb`.